wordpress-mobile/gutenberg-mobile
# e2e test update ios simulators versions

This PR just updates the simulators being used for e2e tests and refactor the code a bit to make clear what simulators are used in CI and the one used locally.

To test:
 - Run the iOS e2e tests locally: `yarn test:e2e:ios:local`
 - Check if all is ok.

PR submission checklist:
